Src/library/wel/windows/wel_window.e

```diff
@@ -1004,16 +1006,22 @@ feature -- Basic operations
 			-- Move the window to `a_x', `a_y'.
 		require
 			exists: exists
+		local
+			l_dpi: INTEGER
 		do
-			move_and_resize_internal (a_x, a_y, 0, 0, True, Swp_nosize)
+			l_dpi := cwin_get_dpi_for_window (item)
+			move_and_resize_internal (cwin_mul_div (a_x, l_dpi, 96), cwin_mul_div (a_y, l_dpi, 96), cwin_mul_div (0, l_dpi, 96), cwin_mul_div (0, l_dpi, 96), True, Swp_nosize)
```

Comment:
It would make sense to replace 96 with something more descriptive. Also, it would make sense to explain which kind of DPI_AWARENESS is in effect.